### PR TITLE
New Rule: Added no-continue core rule

### PR DESCRIPTION
**Pull request by Cisplatin**
As requested on https://augur.net/bounties/.

Please let me know if there is anything you can recommend changing to this PR.

### CODE REVIEW

**Comment by bwitt2** [comment]
As these new rules are added the number of libraries listed in the rules.js test is going to need to be incremented to 21 instead of 20 (and so forth). This may apply to this pull request or to previous ones.

**Comment by beaugunderson** [comment]
i think the augur bounty rules are probably better as PRs on https://github.com/duaraghav8/solium-plugin-security

File: lib/rules/no-continue.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	},
+
+	create: function (context) {
+
+		function inspectContinueStatement (emitted) {
+			if (emitted.exit) { return; }
+
+			context.report ({
+				node: emitted.node,
+				message: 'Use of \'continue\' statement.'
```
Comment (duaraghav8) [inline lib/rules/no-continue.js]: Please change it to `Avoid use of 'continue' statement.`

**Comment by duaraghav8** [comment]
Thanks @Cisplatin! As mentioned by Beau, could you please redirect this PR to  https://github.com/duaraghav8/solium-plugin-security

Implementing & testing rules in plugins is same as that in core linter, so no major disruptions.
See [working with plugins](http://solium.readthedocs.io/en/latest/developer-guide.html#developing-a-plugin).

Feel free to reach out at duaraghav8@gmail.com or https://gitter.im/Solium-linter/Lobby#

**Comment by Cisplatin** [comment]
Moved to https://github.com/duaraghav8/solium-plugin-security/pull/2 as requested.

**Pull request closed without merging**